sample_project/Assets/SampleViewer/Samples/Routing/RouteManager.cs

```diff
@@ -148,12 +149,17 @@ void Start()
         lastRootPosition = arcGISMapComponent.GetComponent<HPRoot>().RootUniversePosition;
     }
 
-    /// <summary>
-    /// Return GeoPosition Based on RaycastHit; I.E. Where the user clicked in the Scene.
-    /// </summary>
-    /// <param name="hit"></param>
-    /// <returns></returns>
-    private ArcGISPoint HitToGeoPosition(RaycastHit hit, float yOffset = 0)
+	private void Update()
+	{
+        RebaseRoute();
+	}
```

Comment:
Please limit to just the functional changes. No need to change other formatting in this PR.